ark/mibi/qc_comp_test.py

```diff
@@ -22,6 +39,117 @@
     (['fov0', 'fov1'], ['chan0', 'chan1'], True)
 ]
 
+MIBITRACKER_EMAIL = 'qc.mibi@gmail.com'
+MIBITRACKER_PASSWORD = 'The_MIBI_Is_Down_Again1!?'
```

Comment:
*cough *cough *cough (set these as env variables on the travis server; otherwise anyone can just see this and use the qc mibitracker account)

also, let's change that pw now lol

Reply:
@ngreenwald can we add these to the Travis server as global environment variables?

Reply:
@ackagel also, have you changed the `MIBITRACKER_PASSWORD`?

Reply:
UPDATE: we've changed `qc.mibi@gmail.com` to be read-only and have access to just Jaiswal's project, resolving.